PWGLF/Tasks/Resonances/doubleResonanceScan.cxx

```diff
@@ -0,0 +1,548 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \file doubleResonanceScan.cxx
+/// \brief Resonance Scanner with ResoTracks and ResoMicroTracks
+/// \author Bong-Hwi Lim <bong-hwi.lim@cern.ch>
+/// \since 27/03/2025
+///
+
+#include <vector>
+#include <TLorentzVector.h>
```

Comment:
`TLorentzVector` is a legacy class and should not be used.

Reply:
Thanks for the suggestion, I'd try with `ROOT::Math::LorentzVector`

Reply:
I temporarily reverted to use the original method. To use the other method, it needs a time to apply and test.